### ISSUE TITLE
Error reading alluxio-env config: line doesn't match format

### ISSUE THREAD

**Issue by dbw9580** [Alluxio/alluxio#18447]
**Alluxio Version:**
latest main

**Describe the bug**
Alluxio CLI throws error when debugging options are set in `alluxio-env.sh`.

```
[alluxio@Fuse3MultiUfs-workers-0 alluxio]$ ./bin/alluxio info nodes
Error: error defining alluxio environment
 --- at /private/var/folders/6d/wq5g449n5nvgt2mf_xbt1sy00000gn/T/alluxio880469490/cli/src/alluxio.org/cli/launch/launch.go:57 (Launcher.GetPreRunFunc.func1) ---
Caused by: error reading alluxio-env config
 --- at /private/var/folders/6d/wq5g449n5nvgt2mf_xbt1sy00000gn/T/alluxio880469490/cli/src/alluxio.org/cli/env/env.go:87 (InitAlluxioEnv) ---
Caused by: While parsing config: line `ALLUXIO_FUSE_ATTACH_OPTS+=" -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=60009"` doesn't match format
Usage:
```

**To Reproduce**

1. Set some environment variable in `alluxio-env.sh`, e.g. `ALLUXIO_FUSE_ATTACH_OPTS+=" -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=60009"`
2. Run `./bin/alluxio info nodes` or `./bin/alluxio info version`

**Expected behavior**
No error is expected.

**Urgency**
Normal.

**Are you planning to fix it**
No.

**Additional context**

**Comment by Xenorith** [comment]
i'm guessing the issue is because the variable is being appended with `+=` instead of being set with `=`
have you tried to directly set `ALLUXIO_FUSE_ATTACH_OPTS=` instead of appending with `+=`?

unfortunately the error is thrown within the parsing library and not within the code within the codebase, so there may not be a straightforward fix for this

**Comment by dbw9580** [comment]
> the issue is because the variable is being appended with += instead of being set with =

Yes, replacing `+=` with `=` won't cause the error to occur. 

I took a look at the code parsing the env file. It's using the [viper](https://github.com/spf13/viper) library, which looks like a parser for config files of different kinds. I presume it cannot parse arbitrary shell syntax?

**Comment by dbw9580** [comment]
> there may not be a straightforward fix for this

I agree. If we want to continue to treat `alluxio-env.sh` as a real shell script, then we may need to actually run it with a shell interpreter within the subprocess launched by the Go launcher, and then run the alluxio java jars. Otherwise, I think we should deprecate `alluxio-env.sh` and replace it with a proper config file, and treat the debug options as a config item, not as a environment variable that can be arbitrarily manipulated.

**Comment by Xenorith** [comment]
hmm your comment gives me the idea that we could run the script and then inspect the state of the environment variables and build the `viper` object manually. this way we can still use `viper` to handle the rest of the setting/updating variable logic as well as keep the `alluxio-env.sh` as a shell script
